### PR TITLE
renamed cubical library

### PR DESCRIPTION
**Pull request by Alizter**
Renamed Square and Cube to PathSquare and PathCube respectively. We have a library of squares in development in the wildcat branch currently so this is in preparation for that. This closes mikeshulman/HoTT#38.

### CODE REVIEW

**Comment by Alizter** [comment]
It looks like proviola is acting up again...

**Comment by Alizter** [comment]
Proviola really doesn't like my cubical stuff. This is related to #1072. @JasonGross any idea what is going on here?

**Comment by JasonGross** [comment]
Nope, sorry.  Presumably you're near some edge case and small changes risk hitting it?  If you can discover the root cause, that'd be great.

File: .travis.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ env:
    - WITH_AUTORECONF="yes" UPDATE_HTML="" UPDATE_QUICK_DOC="" UPDATE_AUTOGEN="" UPDATE_DEP_GRAPHS="" BUILD_COQ="" FORCE_COQ_VERSION="" VALIDATE=""
    - UPDATE_DEP_GRAPHS="yes" WITH_AUTORECONF="yes" BUILD_COQ="yes"
    - UPDATE_QUICK_DOC="yes"  WITH_AUTORECONF="yes" UPDATE_AUTOGEN="yes"
-   - UPDATE_HTML="yes"       WITH_AUTORECONF="yes"
+#   - UPDATE_HTML="yes"       WITH_AUTORECONF="yes"
```
Comment (JasonGross) [inline .travis.yml]: No, if you just do this, then we don't get any html documentation.  If you're going to disable proviola, then just disable proviola, not all of the html doc

Comment (mikeshulman) [reply .travis.yml]: Ok, then you do it.  I don't have the time to figure out how that works, but I want Travis to run on our PRs.

**Comment by JasonGross** [comment]
I don't think there's actually any issue with proviola here.  If you look at the logs, you'll see
```
Hit:32 http://ppa.launchpad.net/couchdb/stable/ubuntu trusty Release
Fetched 27.2 kB in 1s (15.2 kB/s)
Reading package lists... Done
W: GPG error: http://dl.google.com/linux/chrome/deb stable Release: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 78BD65473CB3BD13
W: The repository 'http://dl.google.com/linux/chrome/deb stable Release' is not signed.
N: Data from such a repository can't be authenticated and is therefore potentially dangerous to use.
N: See apt-secure(8) manpage for repository creation and user configuration details.
W: There is no public key available for the following key IDs:
78BD65473CB3BD13  
W: GPG error: https://packagecloud.io/github/git-lfs/ubuntu trusty InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 6B05F25D762E3157
W: The repository 'https://packagecloud.io/github/git-lfs/ubuntu trusty InRelease' is not signed.
N: Data from such a repository can't be authenticated and is therefore potentially dangerous to use.
N: See apt-secure(8) manpage for repository creation and user configuration details.
W: There is no public key available for the following key IDs:
6B05F25D762E3157  
W: http://ppa.launchpad.net/couchdb/stable/ubuntu/dists/trusty/Release.gpg: Signature by key 15866BAFD9BCC4F3C1E0DFC7D69548E1C17EAB57 uses weak digest algorithm (SHA1)
```
I tried running a copy of this on my own fork, and everything worked fine.  If you revert the change to disable proviola and run it again, it might just work.

Maybe we should make this a fatal error?

**Comment by JasonGross** [comment]
Oh, wait, sorry, my fork hasn't finished running yet, so my diagnosis might be wrong.  I'll report back when it finishes.

**Comment by Alizter** [comment]
Excellent! Thank you very much @JasonGross for taking a look at this. Everything seems to be smooth for now.

Getting a lot of these errors in the travis log for the html job, but no harm seems to be done.
```
Exception AttributeError: "'Popen_async' object has no attribute 'terminate'" in <bound method Coq_Local.__del__ of <coq_local.Coq_Local object at 0x2b80f8bc8a10>> ignored
Exception in thread Thread-2 (most likely raised during interpreter shutdown):
Traceback (most recent call last):
  File "/opt/python/2.7.14/lib/python2.7/threading.py", line 801, in __bootstrap_inner
  File "/opt/python/2.7.14/lib/python2.7/threading.py", line 754, in run
  File "/home/travis/build/HoTT/HoTT/proviola/camera/Popen_noblock.py", line 20, in enqueue_output
  File "/opt/python/2.7.14/lib/python2.7/Queue.py", line 138, in put
  File "/opt/python/2.7.14/lib/python2.7/threading.py", line 384, in notify
<type 'exceptions.TypeError'>: 'NoneType' object is not callable
```

**Comment by JasonGross** [comment]
> Getting a lot of these errors in the travis log for the html job, but no harm seems to be done.

Ah, thanks, should be fixed now

**Comment by Alizter** [comment]
@JasonGross 

> Ah, thanks, should be fixed now

Did you manage to fix the errors from coming up? I don't think that change made it in here.

**Comment by JasonGross** [comment]
The fix was upstream in the proviola repo

**Pull request merged**